pkg/api/v1beta3/dynakube/telemetryservice_props.go

```diff
@@ -0,0 +1,33 @@
+package dynakube
+
+const (
+	TelemetryServiceOtlpProtocol   = "otlp"
```

Comment:
you can use golang-like enums (similar what k8s does)
```
type Protocol string

type Protocols []Protocol

const (
        JagerProtocol  Protocol = "jager"
        ZipkinProtocol Protocol = "zipkin"
        OtlpProtocol   Protocol = "otlp"
        StatsdProtocol Protocol = "statsd"
)
```

Reply:
or https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L1524-L1534